charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -27,6 +27,10 @@ spec:
             - name: http
               containerPort: 8000
           env:
+{{- range $key, $value := .Values.env }}
+            - name: {{ $key | replace "." "_" }}
```

Comment:
Even when sticking to this approach (cfr. https://github.com/scality/Zenko/pull/121/files#r193228860), this is wrong. When one sets the key to something with a dot, this may have some reason. If it should be an `_`, one should use an `_` in the values definition. Rewriting this behind the user's back seems odd, and can easily cause confusion.

Reply:
It's just a protection since most shells don't support dots in env variables. Found it on some other Kubernetes charts: https://github.com/kubernetes/charts/blob/master/incubator/zookeeper/templates/statefulset.yaml#L69